docker-for-mac/troubleshoot.md

```diff
@@ -352,3 +353,91 @@ in the Apple documentation, and Docker Desktop [Mac system requirements](install
   sudo launchctl unload /System/Library/LaunchDaemons/org.ntp.ntpd.plist
   sudo launchctl load /System/Library/LaunchDaemons/org.ntp.ntpd.plist
   ```
+
+## Support
+
+This section contains information on how to get support on Docker Desktop.
+
+>**Note**
+>
+> Docker Desktop offers support for Pro and Team users only. For more information about the pricing plans, see [Docker Pricing](https://www.docker.com/pricing){: target="_blank" rel="noopener" class="_"}.
+
+### How do I get Docker Desktop support?
+
+If you have subscribed to a Pro and Team account, please raise a ticket through [Docker Desktop support](https://hub.docker.com/support/desktop/){:target="_blank" rel="noopener" class="_"}.
+
+Docker Community users can get support through our Github repos for-win and for-mac, where we respond on a best-effort basis.
+
+### What support can I get?
+
+If you are a Pro or a Team user, you can request for support on the following types of issues:
+
+* Desktop upgrade issues
+* Desktop installation issues
+  * Enabling virtualization in BIOS
+  * Enabling Windows features
+  * Genuine installation crashes
```

Comment:
Agree. I'll remove 'Genuine' as I'm not sure we can provide more context here on how to differentiate between genuine and fake crashes. We can consider adding another entry if we want to exclude certain types of crashes. However, I'm not sure what they look like atm.

Reply:
Right. Considering we are removing all the sub-lists under the Installation issues, not sure if retaining just one item (Installation crashes) adds any value. I'm tempted to remove this as well and just keep the entry 'Installation issues'. Let me know if you think otherwise,